apollo.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   client: {
     service: {
       name: 'ocean',
-      url: 'https://subgraph.rinkeby.oceanprotocol.com/subgraphs/name/oceanprotocol/ocean-subgraph',
+      url: 'https://subgraphv4.rinkeby.oceanprotocol.com/subgraphs/name/oceanprotocol/ocean-subgraph',
```

Comment:
oh boy, this is so tricky cause we ideally we now continue to have working test runs and builds but until we refactor all subgraph stuff for new subgraph, generating typings from the new subgraph should be postponed. So would suggest putting this back to the old Rinkeby subgraph, I just hope this doesn't affect the Aquarius part.

The goal is that e.g. locally for you `npm test` runs successfully.

Reply:
Sorry, I pushed it before I saw[ your comment](https://github.com/oceanprotocol/market/issues/968#issuecomment-1010979210). sure, I'll put back the old subgraph and only work on having 'Recently Published' section working for now.